pkg/apis/aws/types_infrastructure.go

```diff
@@ -32,6 +32,11 @@ type InfrastructureConfig struct {
 	// See https://registry.terraform.io/providers/hashicorp/aws/latest/docs/guides/resource-tagging#ignoring-changes-in-all-resources
 	// for details of the underlying terraform implementation.
 	IgnoreTags *IgnoreTags
+
+	// EnableCsiEfs enables CSI EFS driver
+	// infra will add additional security group in bound rules and create an amazon EFS file system
+	// Defaults to false
+	EnableCsiEfs *bool
```

Comment:
probably this can be renamed to `filestore` or something more closely related to the actual resource.